src/mpi/coll/src/coll_impl.c

```diff
@@ -18,16 +18,17 @@
 cvars:
     - name        : MPIR_CVAR_DEVICE_COLLECTIVES
       category    : COLLECTIVE
-      type        : boolean
-      default     : true
+      type        : enum
+      default     : percoll
       class       : none
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_ALL_EQ
-      description : >-
-        If set to true, MPI collectives will allow the device to override
-        the MPIR-level collective algorithms. The device still has the
-        option to call the MPIR-level algorithms manually.  If set to false,
-        the device-level collective function will not be called.
+      description : |-
+        Variable to select whether the device can override the
+        collective algorithm.
```

Comment:
...override the *MPIR-level* collective algorithm*s*.

Reply:
Fixed.